installing/installation-guide.md

```diff
@@ -31,8 +31,16 @@ Furthermore, if you are installing Qubes on a potentially compromised system, we
 Qubes 4.0.1 Warning
 -------------------
 
-After installing Qubes 4.0.1, please install fresh APT-based (e.g., Debian and Whonix) TemplateVMs in order to be protected from the APT update mechanism vulnerability that was patched after the release of 4.0.1.
-For instructions and further details, please see [QSB #46].
+Immediately after installing Qubes 4.0.1, please upgrade all of your APT-based (e.g., Debian and Whonix) TemplateVMs by executing the following command in a dom0 terminal for each such TemplateVM:
+
+    $ sudo qubes-dom0-update --action=upgrade <template-package-name>
```

Comment:
Should it also mention relevant --enablerepo option for Whonix templates?

Reply:
How about how?